Functions/xASL_bids_BIDS2Legacy.m

```diff
@@ -250,17 +250,27 @@ function xASL_bids_BIDS2Legacy(pathStudy, bOverwrite, dataPar)
 
 %% 9. Parse M0
 ListASL4D = xASL_adm_GetFileList(pathLegacy, '^ASL4D\.nii$', 'FPListRec');
-if ~isempty(ListASL4D)
-    for iList=1:numel(ListASL4D)
-        xASL_bids_parseM0(ListASL4D{iList});
+try
```

Comment:
No longer needed. Good that you had it here for testing, but lets remove it now.

Reply:
Removed it here b1758a0.